openedx/features/pakx/lms/pakx_admin_app/serializers.py

```diff
@@ -129,19 +131,27 @@ def get_role(obj):
 class UserProfileSerializer(serializers.ModelSerializer):
     language_code = LanguageProficiencySerializer(write_only=True)
     languages = serializers.SerializerMethodField(read_only=True)
+    name = serializers.CharField(required=True)
 
     class Meta:
         model = UserProfile
         fields = ('name', 'employee_id', 'languages', 'language_code', 'organization')
 
     def validate_name(self, value):
-        if not self.instance and not value.strip():
-            raise serializers.ValidationError('This field required!')
+        if not value.strip():
+            raise serializers.ValidationError('User\'s name is required')
+        if not re.match('^[a-zA-Z ]+$', value):
+            raise serializers.ValidationError('Only alphabets are allowed')
         return value
 
     def validate_organization(self, value):
-        if not self.instance and not value:
-            raise serializers.ValidationError('This field required!')
+        if not value:
+            raise serializers.ValidationError('Organization can\'t be empty')
+        return value
+
+    def validate_employee_id(self, value):
+        if value and not re.match('^[A-Za-z0-9-_]+$', value):
```

Comment:
can we define a helper function that takes a pattern and value and return the flag, we are using this above ☝🏽  as well.

Reply:
that would be just a wrapper function looking like this `re.match(pattern, text)` which will be almost the same as now. creating a util function for this will cost us an extra function call and not serve the purpose of the DRY principle